### ISSUE TITLE
Import crash related to #113

### ISSUE THREAD

**Issue by jhhcs** [JarryShaw/PyPCAPKit#114]
Unfortunately, the fix for #113 seems to have introduced another issue that appears when PyPCAPKit is used in a Python 3.7 environment. The following is on Python 3.7.0, Windows 10, CPython:
```
Python 3.7.0 (v3.7.0:1bf9cc5093, Jun 27 2018, 04:59:51) [MSC v.1914 64 bit (AMD64)]
Type 'copyright', 'credits' or 'license' for more information
IPython 7.34.0 -- An enhanced Interactive Python. Type '?' for help.

In [1]: import pcapkit
Traceback (most recent call last):

  File "X:\test\venv\lib\site-packages\IPython\core\interactiveshell.py", line 3553, in run_code
    exec(code_obj, self.user_global_ns, self.user_ns)

  File "<ipython-input-2-fdafce80c9fe>", line 1, in <module>
    import pcapkit

  File "X:\test\venv\lib\site-packages\pcapkit\__init__.py", line 83, in <module>
    from pcapkit.foundation.registry import *

  File "X:\test\venv\lib\site-packages\pcapkit\foundation\__init__.py", line 13, in <module>
    from pcapkit.foundation.extraction import Extractor

  File "X:\test\venv\lib\site-packages\pcapkit\foundation\extraction.py", line 424
    if (parent := os.path.split(fout)[0]):
               ^
SyntaxError: invalid syntax
```

**Comment by JarryShaw** [comment]
I'm still working on a solution to resolve the installation time back-compiling integration with [`pybpc`](https://github.com/pybpc) and `setuptools`. At the moment, please consider using the following steps to resolve the issue:

1. install `python-walrus` into the same virtual environment
2. run `walrus --no-archive X:\test\venv\lib\site-packages\pcapkit` in the virtual environment

This will manually convert the assignment expressions used in the project and make it compliant with older grammars.

**Comment by JarryShaw** [comment]
Should've been fixed in version 0.16.3
